### DISCUSSION TITLE
Customize public assets cache max-age

### DISCUSSION THREAD

**Discussion by xolott** [meteor/meteor#12745] [Feature Requests]
Looking at [the code](https://github.com/georgyberdyshev/meteor/blob/678b7b9bdc7042a875d2c9af07626f99ba636ff7/packages/webapp/webapp_server.js#L434) we don't have a way to customize the max-age of the public assets. This was discussed multiple times before, but instead of forcing only `max-age=0` to everyone, I strongly believe this should be customizable. 

I have assets that never change (for the purpose of our project), and I want to use our CDN to cache those files. But since they always return `max-age=0` it never happens.

**Comment by harryadel** [comment]
This looks like a prime candidate for hacktoberfest pull request. I suggest that you create a PR where you can set it using an environment variable or something similar. What do you guys think? cc @StorytellerCZ @Grubba27
